pom.xml

```diff
@@ -115,6 +122,8 @@
                         <failIfNoTests>false</failIfNoTests>
                         <systemPropertyVariables>
                             <java.io.tmpdir>${project.build.directory}</java.io.tmpdir>
+                            <test.instance.name>${test.instance.name}</test.instance.name>
+                            <azure.group>${azure.group}</azure.group>
```

Comment:
IMO it is not appropriate to define module specific property in root pom. Can you move the logic into module's poms? Also timestamp properties.